examples/ITR_UI.py

```diff
@@ -1768,10 +1808,12 @@ def reset_filters(n_clicks_reset):
     if (ProjectionControls.TREND_CALC_METHOD != ITR_median
         or ProjectionControls.LOWER_PERCENTILE != 0.1
         or ProjectionControls.UPPER_PERCENTILE != 0.9
-        or ProjectionControls.TARGET_YEAR != 2050):
+        or ProjectionControls.TARGET_YEAR != 2050
+        or ProjectionControls.TARGET_PROBABILITY != 0.5):
```

Comment:
Per the first comment, we need to figure out where this value should live--as part of the TemperatureScore initialization or as part of ProjectionControls (which TemperatureScore would read).  I think it's cleaner to put it in TemperatureScore, but an argument could be made to put it in ProjectionControls and read it from TemperatureScore.  It's worthy of thought/discssion.

Reply:
The target probability is now part of the class TempartureScoreConfig.